cypress/integration/login-test.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable no-undef */
+const EMAIL = '';
+const PASSWORD = '';
+
+const logIn = () => {
+  cy.get(".login-button")
+    .click();
+  cy.get(".form > .input-group:nth-child(1) > input")
+    .type(EMAIL);
+  cy.get(".form > .input-group:nth-child(2) > input")
+    .type(PASSWORD);
+  cy.get('.form > .btn')
+    .click();
+}
+
+const logOut = () => {
+  cy.get('.navbar-link-profile')
+    .click();
+  cy.get(".popover-body > .profile-modal-container > input:nth-child(7)")
```

Comment:
Didn't check actual HTML for elements, but might be desired to assign unique `id` to elements to avoid hardcoding full paths where avoidable --  so tests would become more "semantically driven" and less fragile to code changes.

Reply:
Strong disagree on using id’s. Super nom-react’y way of approaching things, often causes more problems than it solves.

Our newer patterns of class naming (following BEM patterns) gives us ‘component name paths’ that can be used in most cases.

I haven’t looked at the specific paths here, but they may also be able to be optimised further.